lib/active_merchant/billing/gateways/checkout_v2.rb

```diff
@@ -138,6 +139,26 @@ def add_transaction_data(post, options = {})
         post[:previous_payment_id] = options[:previous_charge_id] if options[:previous_charge_id]
       end
 
+      def add_stored_credential_options(post, options = {})
+        return unless options[:stored_credential]
+
+        case options[:stored_credential][:initial_transaction]
+        when true
+          post[:merchant_initiated] = false
+        when false
+          post[:'source.stored'] = true
```

Comment:
Is this meant to have quotes? `'source.stored'` ?

Reply:
Yep! That is the name of the field according to [checkout's docs](https://docs.checkout.com/quickstart/use-an-existing-card/stored-card-details) and without the quotes, tests fail because they search for a method called 'stored'.

Reply:
Wonder if it's because it uses a `.` instead of a `_` to break up the words